auto_revoke_sg_rules/iam.tf

```diff
@@ -39,7 +39,7 @@ resource "aws_iam_role_policy" "security_group_modification" {
         ]
         Effect   = "Allow"
         Sid      = "AllowSnsActions"
-        Resource = "arn:aws:sns:${local.region}:${local.account_id}:internal-sre-alert"
+        Resource = "arn:aws:sns:${local.region}:${local.account_id}:${var.sns_topic}"
```

Comment:
Instead of building up an arn just accept the arn as the input.

Reply:
Thanks Calvin! Sorry for the basic question, but how would this work though given that we are applying it all accounts and we don't know the arn information when we are applying it [here](https://github.com/cds-snc/aft-global-customizations/blob/main/terraform/auto_revoke_sg_rules.tf)? Would you suggest creating a local block in the global customizations module something similar to the code below?:
```
locals {
  sns_topic_arn = "arn:aws:sns:${local.region}:${local.account_id}:internal-sre-alert.arn"
}

module "auto_revoke_sg_rules" {
  source            = "github.com/cds-snc/terraform-modules?ref=v5.0.3//auto_revoke_sg_rules"
  billing_tag_value = "SRE"
  sns_topic_arn  = sns_topic_arn
}
```

Reply:
So instead of using the data lookups for data and region within this module the code that calls it will do that and build the arn that way. 

I was only thinking of passing an arn because we can then modify in the future to pass multiple arns if we want to alert to multiple things. 

But we can always change that in the future so I'll approve as is.